src/seedu/addressbook/commands/ListFriendCommand.java

```diff
@@ -0,0 +1,42 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.tag.Tag;
+import seedu.addressbook.data.tag.UniqueTagList;
+
+import java.util.*;
+
+public class ListFriendCommand extends Command {
```

Comment:
what if you need to list people with other tag?

Reply:
Hi, can I ask about the 1kLoc enhancement? is it because I did not submit a PR which caused it to be not recorded? As during that tutorial session, I only showed you the code but did not create a PR.

Reply:
In addition, can I also ask about the CI failure, as when i review the differences, they are exactly the same

Reply:
for the 1kLoC, you were required to update all 3 parts, code, tests and docs in order to receive credits. 

I believe they are because of [whitespaces](https://github.com/nus-cs2103-AY1718S2/addressbook-level2/pull/590/files#diff-d2155875e4d28e1f0aa98b86e0ff79d8L92)

Reply:
alright, noted. Thank you for your reply. will submit the second enhancement before the next tutorial